doc/i18n.md

```diff
@@ -54,19 +54,19 @@ Translation process
    Running `make po/cockpit.pot` will generate a standard PO template.
 
  * With `make upload-pot` the PO template gets uploaded to the
-   [Zanata translation platform](https://fedora.zanata.org/iteration/view/cockpit/master)
+   [Weblate translation platform](https://translate.stg.fedoraproject.org/projects/cockpit/)
    where everybody can contribute translations to various languages.
 
- *  With `make download-po` Zanata's translations are downloaded to
+ *  With `make download-po` Weblate's translations are downloaded to
     `po/XX.po`. This is done by
    [bots/po-refresh](https://github.com/cockpit-project/cockpit/blob/master/bots/po-refresh)
    which is invoked regularly by
    [bots/po-trigger](https://github.com/cockpit-project/cockpit/blob/master/bots/po-trigger).
 
  * Translations can also be done using any other tool or platform and be
    submitted via pull requests that update the `po/XX.po` files. In this case
-   they should then also be uploaded to Zanata again with
-   `make upload-translations`.
+   they should then also be uploaded to Weblate either manually using Weblate UI
+   or by pushing these files into [cockpit-weblate](https://github.com/cockpit-project/cockpit-weblate) repository.
```

Comment:
Interesting, these will automatically be imported into weblate then? So this is really a bi-directional data store also for .po, not just for the .pot?

Reply:
It can cause conflicts. But when properly executed, it would work. Therefore I don't want any PRs into our weblate repos, as it requires some manual work.
(I don't know what exactly would be needed, but in weblate UI there are actions like 'Push into remote repo local changes`, `Pull changes from remote repo`, `Rebase onto remote repo`... so when done in the right order, there would not be any conflicts).